java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/BUILD.bazel

```diff
@@ -1,58 +1,46 @@
 load("@rules_java//java:defs.bzl", "java_binary", "java_library")
 load("//java/private:artifact.bzl", "artifact")
 
-# We want to be able to compile the test runner, but we don't want to
-# expose any of its dependencies, since we expect whoever is running
-# tests to add them themselves, and we don't want to pollute the classpath.
-# As such, we do this rather weird thing to make sure we have the runner
-# but none of its dependencies.
-java_import(
```

Comment:
This was done deliberately so that we don't leak any of our dependencies into people's builds. We want the user to be bringing their version of junit to the table, not accidentally picking up our version